src/main/java/com/juniorstart/juniorstart/service/JobOfferService.java

```diff
@@ -0,0 +1,79 @@
+package com.juniorstart.juniorstart.service;
+
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import com.juniorstart.juniorstart.exception.BadRequestException;
+import com.juniorstart.juniorstart.model.JobOffer;
+import com.juniorstart.juniorstart.model.User;
+import com.juniorstart.juniorstart.payload.ApiResponse;
+import com.juniorstart.juniorstart.payload.JobOfferRequest;
+import com.juniorstart.juniorstart.repository.JobOfferRepository;
+import com.juniorstart.juniorstart.repository.UserDao;
+
+@Service
+public class JobOfferService
+{
+
+	JobOfferRepository jobOfferRepository;
```

Comment:
Brakuje private final żeby nie dało się bezpośrednio do repozytorium odwoływać, bez funkcji pośrednich, które mógłby na przykład walidować dane

Reply:
to samo co na górze